wrapper/telemetry-wrapper.js

```diff
@@ -353,14 +352,14 @@ window.TelemetryWrapper.go = function (params, element) {
     var percentileLabel = ' - medians'; // i18n?
     var valuesArePercent = false;
     if (kind == 'enumerated' || kind == 'boolean' || kind == 'flag') {
-      const BUCKET_INDEX_FOR_ENUMERATED = 0;
+      const bucketIndex = params.evoBucketIndex > 0 ? params.evoBucketIndex : 0;
       if (kind == 'boolean') {
         yLabel = desc + ' % FALSE'; // TODO: i18n
```

Comment:
Ohhh... here's an interesting case. What if the evoBucketIndex is set to 1 for a boolean histogram? Looks like it'll claim it's "% FALSE" even if we tell it to plot the TRUE bucket.

Reply:
Hm, come to think of it... it looks like this should be used for kind == 'boolean' or kind == 'flag'.